src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/CNNVariantWriteTensors.java

```diff
@@ -47,22 +47,22 @@
  * gatk CNNVariantWriteTensors \
  *   -R reference.fasta \
  *   -V input.vcf.gz \
- *   -truth-vcf platinum-genomes.vcf \
- *   -truth-bed platinum-confident-region.bed \
- *   -tensor-name reference \
- *   -output-tensor-dir my-tensor-folder
+ *   --truth-vcf platinum-genomes.vcf \
+ *   --truth-bed platinum-confident-region.bed \
+ *   --tensor-type reference \
```

Comment:
There is another reference to `tensor-name` in the documentation above here and in CNNVariantTrain which would be good to update as well.